Cargo.toml

```diff
@@ -14,7 +14,7 @@ exclude = [".github/workflows", "tarp.sh"]
 
 
 [dependencies]
-bpaf_derive = { path = "./bpaf_derive", version = "0.2.2", optional = true }
+bpaf_derive = { path = "./bpaf_derive", version = "0.3.0", optional = true }
```

Comment:
For the longest time I used `^` (the default constraint operator) but I've given in and followed serde/serde_derive and use `=`.  The problem is that `bpaf` and `bpaf_derive` depend on each other in practice and getting them out of sync can break people, so its best to just enforce that they are in lockstep with each other.

Reply:
> I guess you are right. In practice they don't depend on each other nearly as much as serde and so far any matching pair would just work,

You sure? If you add a new feature to` bpaf` and need to change `bpaf_derive` for it, you've now broken people.

`bpaf_derive` makes direct reference to a lot of `bpaf`s API, so this seems likely.

Reply:
At the moment - yes, the API surface is really small. But I already changed it to use `=` just in case.